src/common/utils/constants.ts

```diff
@@ -8,5 +10,7 @@ export const AUTHORIZATION_ERROR_MESSAGE
 export const FORBIDDEN_ERROR_MESSAGE
   = "You are not allowed to perform this action.";
```

Comment:
For these static messages also better to have them as functions for consistency
Eg: () => 'Error message';